pages/summary-confirmation.js

```diff
@@ -149,6 +185,13 @@ const ApplicationComplete = props => (
         Find out here what you can do to prepare:
       </Header>
       <HintText mb={3}>All links open in a new window</HintText>
+
+      {wales.includes(props.currentLA) && (
```

Comment:
It is necessary to define test cases for each country to meet the test coverage threshold of 100%